### PR TITLE
53 redesign initial screens

### LINKED ISSUE #53: DESIGN: Redesign Welcome, Login, And Create Account

### PR DESCRIPTION
**Pull request by JeffKandel**
closes #53

### CODE REVIEW

**Review by dffattal — COMMENTED** [review]
Design looks really good. Indentation could be a bit cleaner (indent nested react components), and there are a couple of logs left in. Could we also add our constatns/dev.js to .gitignore?

File: app/redux/reducers/users.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ import { setViewerGroups } from './groups'
 import { ipAddress, port} from '../../../constants/dev'
 
 export const createAccount = user => dispatch => {
+    console.log("user", user);
```
Comment (miggitymack) [inline app/redux/reducers/users.js]: take out console.log

**Review by miggitymack — DISMISSED** [review]
Looks good. I'll fix indenting.

**Review by miggitymack — APPROVED** [review]
Last minute deletions

**Pull request merged**